client/src/components/NetWorth.tsx

```diff
@@ -0,0 +1,163 @@
+import React from 'react';
+import IconButton from 'plaid-threads/IconButton';
+import Trash from 'plaid-threads/Icons/Trash';
+
+import { currencyFilter, pluralize } from '../util';
+import { Asset } from '.';
+import { AccountType, AssetType } from './types';
+import { useAssets } from '../services';
+interface Props {
+  numOfItems: number;
+  accounts: AccountType[];
+  personalAssets: AssetType[];
+  userId: number;
+  assetsOnly: boolean;
+}
+
+export default function NetWorth(props: Props) {
+  const { deleteAssetByAssetId } = useAssets();
+  // sums of account types
+  const addAllAccounts = (
+    accountSubtypes: Array<AccountType['subtype']>
+  ): number =>
+    props.accounts
+      .filter(a => accountSubtypes.includes(a.subtype))
+      .reduce((acc: number, val: AccountType) => acc + val.current_balance, 0);
+
+  const depository: number = addAllAccounts([
+    'checking',
+    'savings',
+    'cd',
+    'money market',
+  ]);
+  const investment: number = addAllAccounts(['ira', '401k']);
+  const loan: number = addAllAccounts(['student', 'mortgage']);
+  const credit: number = addAllAccounts(['credit card']);
+
+  const personalAssetValue = props.personalAssets.reduce((a, b) => {
+    return a + b.value;
+  }, 0);
+
+  const assets = depository + investment + personalAssetValue;
+  const liabilities = loan + credit;
+
+  const handleDelete = (assetId: number, userId: number) => {
+    deleteAssetByAssetId(assetId, userId);
+  };
+
+  return (
+    <div className="netWorthContainer">
+      <h2 className="netWorthHeading">Net Worth</h2>
+      <h4 className="tableSubHeading">
+        A summary of your assets and liabilities
+      </h4>
+      {!props.assetsOnly && (
+        <>
+          <div className="netWorthText">{`Your total across ${
+            props.numOfItems
+          } bank ${pluralize('account', props.numOfItems)}`}</div>
+          <h2 className="netWorthDollars">
+            {currencyFilter(assets - liabilities)}
+          </h2>
+          <div className="holdingsContainer">
+            <div className="userDataBox">
+              <div className="holdingsList">
+                <div className="assetsHeaderContainer">
+                  <h4 className="dollarsHeading">{currencyFilter(assets)}</h4>
+                  <Asset userId={props.userId} />
+                </div>
+
+                <div className="data">
+                  {/* 3 columns */}
+                  <p className="title">Assets</p>
```

Comment:
I wonder if we should promote this to a title, just to be consistent with the design you use for the cards under "Monthly Spending" (i.e., "Spending Categories" and "Top 5 Vendors" are titles).
<img width="993" alt="Screen Shot 2021-07-15 at 10 01 50 AM" src="https://user-images.githubusercontent.com/86379616/125811838-ad292e18-c7a1-4d04-b8a8-c6ead156290e.png">

Reply:
done